app/src/main/java/org/simple/clinic/editpatient/EditPatientScreen.kt

```diff
@@ -457,6 +457,10 @@ class EditPatientScreen : BaseScreen<
     bpPassportsLabel.visibility = VISIBLE
   }
 
+  override fun showBpPassportLabelIfIdentifierNotEmpty(identifiers: List<String>) {
```

Comment:
Instead of passing all the identifies list to toggle the visibility. Can we just have functions to hide and show the label, please? 

We already have the show label function from the last commit, we can have the hide label function as well and then have the logic to toggle between these two in the UI renderer.

Reply:
Ah this makes more sense, updating this